shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/natived/builder/dialect/OraclePrivilegeHandler.java

```diff
@@ -151,12 +174,12 @@ private void collectSysPrivileges(final Map<ShardingSphereUser, List<PrivilegeTy
     }
     
     private String getSysPrivilegesSQL(final Collection<ShardingSphereUser> users) {
-        String userList = users.stream().map(each -> String.format("'%s'", each.getGrantee().getUsername())).collect(Collectors.joining(", "));
+        String userList = users.stream().map(each -> String.format("%s", each.getGrantee().getUsername())).collect(Collectors.joining(", "));
         return String.format(SYS_PRIVILEGE_SQL, userList);
     }
     
     private String getTablePrivilegesSQL(final Collection<ShardingSphereUser> users) {
-        String userList = users.stream().map(each -> String.format("'%s'", each.getGrantee().getUsername())).collect(Collectors.joining(", "));
+        String userList = users.stream().map(each -> String.format("%s", each.getGrantee().getUsername())).collect(Collectors.joining(", "));
```

Comment:
we can't remove `'` symbol, the target sql is like `xxx in ('admin', 'sys', 'other')`, if you remove `'` the sql will grammar error.